scripts/build_env.sh

```diff
@@ -28,7 +28,7 @@ if [ ! -f "${CUDACXX}" ]; then
    else
       export CMAKE_PREFIX_PATH="${CUDA_HOME}"${CMAKE_PREFIX_PATH:+:${CMAKE_PREFIX_PATH}}
       export PATH="${CUDA_HOME}/bin"${PATH:+:${PATH}}
-      export LD_LIBRARY_PATH="${CUDA_HOME}/lib64"{LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
+      export LD_LIBRARY_PATH="${CUDA_HOME}/lib64"${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
```

Comment:
I wish I saw this earlier. 😭 I spent some minutes trying to find this same mistake...